### PR TITLE
Bukkit Color

### PR DESCRIPTION
**Pull request by UsainSrht**

### CODE REVIEW

**Review by OakLoaf — APPROVED** [review]
This looks good to me!

File: src/main/java/de/oliver/fancyholograms/storage/FlatFileHologramStorage.java [hunk 1]
```diff
@@ -247,14 +247,15 @@ public static HologramData readHologram(String name, ConfigurationSection config
...
             };
 
-            TextColor background = null;
+            Color background = null;
             if (backgroundName != null) {
                 if (backgroundName.equalsIgnoreCase("transparent")) {
                     background = Hologram.TRANSPARENT;
                 } else if (backgroundName.startsWith("#")) {
-                    background = TextColor.fromHexString(backgroundName);
+                    background = Color.fromARGB((int)Long.parseLong(backgroundName.substring(1), 16));
```
Comment (OliverSchlueter) [inline src/main/java/de/oliver/fancyholograms/storage/FlatFileHologramStorage.java:255]: Why do you parse it as long and then cast to integer?

**Review by OliverSchlueter — COMMENTED** [review]
Would be cool, if you could implement it for 1.20.6 too.

File: src/main/java/de/oliver/fancyholograms/storage/FlatFileHologramStorage.java (hunk 1 above)
Comment (UsainSrht) [reply src/main/java/de/oliver/fancyholograms/storage/FlatFileHologramStorage.java:255]: > Why do you parse it as long and then cast to integer?

It was throwing some exception for number being too large or it was something to do with negativity of the number I can't tell right now but you can try with `Integer.parseInt` and see the error yourself.

**Pull request merged**